realms_cli/README.md

```diff
@@ -2,28 +2,30 @@
 
 This is in heavy development and things might not work as expected... Bring a torch to slay some STARK dragons...
 
-## Prerequisites
-
-This feature has been tested in 
+This feature has been tested in
 ```
-python 3.7.12
+python 3.7.13
 ---
 cairo-lang==0.8.1
 cairo-nile==0.6.1
 ```
 
+---
+## Regular Setup
+
+
 <details><summary>Python Setup</summary>
 
 1. Upgrade pip: `/usr/local/bin/python -m pip install --upgrade pip`
```

Comment:
Note: This will [break for OSX users who install via homebrew](https://github.com/Homebrew/legacy-homebrew/issues/26900). The workaround is to upgrade with homebrew: `brew install python3` (or python3@3.7).

Might be worth adding a caveat here.

Reply:
Ok added